packages/admin-stories/src/admin/helpers/DataGenerators.ts

```diff
@@ -0,0 +1,34 @@
+export const generateRandomEmails = (amount: number, valueLength: number) => {
+    return Array.from(Array(amount).keys()).map(() => {
```

Comment:
why not use faker library or something?

Reply:
i needed data so i did it on my own, haven't thought about faker or any other libraries for that :)